polling_stations/apps/api/address.py

```diff
@@ -108,8 +108,15 @@ def retrieve(
         # council object
         ret["council"] = Council.objects.get(pk=address.council)
 
-        # Addresbase object should have a location
-        ret["postcode_location"] = GEOSGeometry(address.location)
+        # attempt to attach point
+        # in this situation, failure to geocode is non-fatal
+        try:
+            l = geocoder(address.postcode)
+            location = l.centroid
+        except PostcodeError:
```

Comment:
I'm not sure this is necessary as the postcodes come from addressbase by definition, so they should always be successfully geocoded by the `AddressbaseGeocoder`. There may be some reason I've not thought of though.